pr-dhl-woocommerce/pr-dhl-woocommerce.php

```diff
@@ -445,6 +456,10 @@ public function get_base_country() {
 		return apply_filters( 'pr_shipping_dhl_base_country', $country_code['country'] );
 	}
 
+    public function get_base_postcode() {
+        return apply_filters( 'pr_shipping_dhl_base_postcode', get_option( 'woocommerce_store_postcode' ) );
```

Comment:
```suggestion
        return apply_filters( 'pr_shipping_dhl_base_postcode', WC()->countries->get_base_postcode() );
```
Instead of using `get_option()` it's better if we use WC native function.